### ISSUE TITLE
[Quest][Netherstorm][Minor issue: quest mechanics] Triangulation Point One

### ISSUE THREAD

**Issue by debix** [FeenixServerProject/Archangel_2.4.3_Bugtracker#192]
Quest Triangulation Point one: http://www.wowhead.com/quest=10269

From wowhead comments:
Make sure that when you use the triangulation device you are not standing in the correct location to begin with (ie: 66., 33.8). Stand a little outside the area and use it. When the mark comes up, walk to it for the quest to complete, otherwise, you can be on the mark all you want, the quest will not show completed.

How it works now (done q 2 times in different ways):
1) No need to use Triangulation Device. Just need to fly over the given coordinates and it gives quest credit automatically.
2) Triangulation device, when used outside coordinates, does not summon "holographic indication" (a hunter mark model) over the needed location. 
So basically The Device has no use

P.S. Apparently this wowhead comment is a bit confusing. The location of The Point itself (one of triangle points. Point, where mark should show up) is 66.9 , 34.2
